server/liveblog/posts/posts.py

```diff
@@ -170,7 +174,7 @@ def on_update(self, updates, original):
         post.update(updates)
         self.check_post_permission(post)
         # when publishing, put the published item from drafts and contributions at the top of the timeline
-        if updates.get('post_status') == 'open' and original.get('post_status') in ('draft', 'submitted'):
+        if updates.get('post_status') == 'open' and original.get('post_status') in ('draft', 'submitted', 'sticky'):
```

Comment:
Do you think that `... original.get('post_status') not 'open':` could work and be more clear here ?